adaptive/runner.py

```diff
@@ -693,6 +700,8 @@ def _get_ncores(ex):
         return 1
     elif with_distributed and isinstance(ex, distributed.cfexecutor.ClientExecutor):
         return sum(n for n in ex._client.ncores().values())
+    elif with_mpi4py and isinstance(ex, mpi4py.futures.MPIPoolExecutor):
+        return mpi4py.MPI.COMM_WORLD.size - 1
```

Comment:
```python
        ex.bootup() # wait until all workers are up and running
        return executor._pool.size  # not public API!
```

Reply:
That's better, does `ex._pool.size` work before all the workers are up and running? Because Adaptive can handle scaling of the pool size.